### ISSUE TITLE
misleading error message (i.e. not an error)

### ISSUE THREAD

**Issue by marianne013** [DIRACGrid/DIRAC#6096]
I'm minding my own business doing some debugging and I am running the multivo test scrips against https://lbcertifdirac70.cern.ch
and I get the error below:

```
(base) lxplus716:8.0.0a21 :~] ./DIRACRepo/tests/System/client_multivo.sh

Testing first VO: gridpp
dirac-login gridpp_user --out /afs/cern.ch/user/d/dbauer/workspace/public/8.0.0a21/gridpp.proxy
2022-05-23 11:11:04 UTC Framework ERROR: No proxy found
2022-05-23 11:11:05 UTC Framework ERROR: Error while updating the configuration Reason(s):
        No proxy found
Enter Certificate password: **********
Upload proxy to server.
[...]
```
It then happily goes on to make my proxy.
Why is it looking for a proxy in a command that is meant to create one ?

**Comment by TaykYoku** [comment]
Hi @marianne013, I'm having trouble adding my certificate to gridpp, so I can't repeat this error. Can you please run:
```
dirac-login gridpp_user --out /afs/cern.ch/user/d/dbauer/workspace/public/8.0.0a21/gridpp.proxy -ddd
```

**Comment by marianne013** [comment]
I don't think it's gridpp related. I just ran the command three times one after another with three different results, maybe the server is upset ?
```
lxplus739:public :~] cd 8.0.0a21/
lxplus739:8.0.0a21 :~] source diracos/diracosrc
(base) lxplus739:8.0.0a21 :~] dirac-login dteam_user --out dteam.proxy -ddd
2022-05-30 12:37:48 UTC Framework [139824217069376] DEBUG: dirac.cfg should be at /afs/cern.ch/work/d/dbauer/public/8.0.0a21/diracos/etc/dirac.cfg
2022-05-30 12:37:48 UTC Framework [139824217069376] DEBUG: CFG merged
2022-05-30 12:37:48 UTC Framework [139824217069376] DEBUG: Updating configuration internals
2022-05-30 12:37:48 UTC Framework [139824217069376] DEBUG: Updating configuration internals
2022-05-30 12:38:03 UTC Framework [139824047597312] DEBUG: Refreshing configuration...
2022-05-30 12:38:03 UTC Framework [139824047597312] DEBUG: Refreshing from list ['https://lbcertifdirac70.cern.ch:9135/Configuration/Server']
2022-05-30 12:38:03 UTC Framework [139824047597312] DEBUG: Randomized server list is https://lbcertifdirac70.cern.ch:9135/Configuration/Server
2022-05-30 12:38:03 UTC Framework [139824047597312] DEBUG:  Trying to refresh from https://lbcertifdirac70.cern.ch:9135/Configuration/Server
2022-05-30 12:38:03 UTC Framework/DIRAC.Core.Tornado.Client.ClientSelector [139824047597312] DEBUG: Trying to autodetect client for https://lbcertifdirac70.cern.ch:9135/Configuration/Server
2022-05-30 12:38:03 UTC Framework/DIRAC.Core.Tornado.Client.ClientSelector [139824047597312] VERBOSE: Using HTTPS for service https://lbcertifdirac70.cern.ch:9135/Configuration/Server
2022-05-30 12:38:03 UTC Framework [139824047597312] DEBUG: Already given a valid url https://lbcertifdirac70.cern.ch:9135/Configuration/Server
2022-05-30 12:38:03 UTC Framework [139824047597312] DEBUG: Already given a valid url https://lbcertifdirac70.cern.ch:9135/Configuration/Server
2022-05-30 12:38:03 UTC Framework [139824047597312] ERROR: No proxy found
2022-05-30 12:38:03 UTC Framework [139824047597312] WARN: Can't update from server Error while updating from https://lbcertifdirac70.cern.ch:9135/Configuration/Server: No proxy found
2022-05-30 12:38:03 UTC Framework [139824047597312] ERROR: Error while updating the configuration Reason(s):
        No proxy found
2022-05-30 12:38:04 UTC Framework [139824217069376] WARN: Could not get NTP time
Parsing command line
Trying to load Resources.LogBackends.StdoutBackend
Trying to load DIRAC.Resources.LogBackends.StdoutBackend
Enter Certificate password: **********
2022-05-30 12:38:14 UTC dirac-login [139824217069376] DEBUG: Trying to load Resources.LogBackends.StdoutBackend
2022-05-30 12:38:14 UTC dirac-login [139824217069376] DEBUG: Trying to load DIRAC.Resources.LogBackends.StdoutBackend
2022-05-30 12:38:14 UTC dirac-login [139824217069376] DEBUG: Refreshing configuration...
2022-05-30 12:38:14 UTC dirac-login [139824217069376] DEBUG: Refreshing from list ['https://lbcertifdirac70.cern.ch:9135/Configuration/Server']
2022-05-30 12:38:14 UTC dirac-login [139824217069376] DEBUG: Randomized server list is https://lbcertifdirac70.cern.ch:9135/Configuration/Server
2022-05-30 12:38:14 UTC dirac-login [139824217069376] DEBUG:  Trying to refresh from https://lbcertifdirac70.cern.ch:9135/Configuration/Server
2022-05-30 12:38:14 UTC dirac-login/DIRAC.Core.Tornado.Client.ClientSelector [139824217069376] DEBUG: Trying to autodetect client for https://lbcertifdirac70.cern.ch:9135/Configuration/Server
2022-05-30 12:38:14 UTC dirac-login/DIRAC.Core.Tornado.Client.ClientSelector [139824217069376] VERBOSE: Using HTTPS for service https://lbcertifdirac70.cern.ch:9135/Configuration/Server
2022-05-30 12:38:14 UTC dirac-login [139824217069376] DEBUG: Already given a valid url https://lbcertifdirac70.cern.ch:9135/Configuration/Server
2022-05-30 12:38:14 UTC dirac-login [139824217069376] DEBUG: Already given a valid url https://lbcertifdirac70.cern.ch:9135/Configuration/Server
2022-05-30 12:38:14 UTC dirac-login [139824217069376] ERROR: No proxy found
2022-05-30 12:38:14 UTC dirac-login [139824217069376] WARN: Can't update from server Error while updating from https://lbcertifdirac70.cern.ch:9135/Configuration/Server: No proxy found
Couldn't generate proxy: Cannot contact CS: Reason(s):
        No proxy found
(base) lxplus739:8.0.0a21 :~] dirac-login dteam_user
2022-05-30 12:38:41 UTC Framework ERROR: No proxy found
2022-05-30 12:38:41 UTC Framework ERROR: Error while updating the configuration Reason(s):
        No proxy found
Enter Certificate password: **********
Upload proxy to server.
subject      : /C=UK/O=eScience/OU=Imperial/L=Physics/CN=daniela bauer/CN=3942386890/CN=3502891271/CN=5705581857
issuer       : /C=UK/O=eScience/OU=Imperial/L=Physics/CN=daniela bauer/CN=3942386890/CN=3502891271
identity     : /C=UK/O=eScience/OU=Imperial/L=Physics/CN=daniela bauer
timeleft     : 11:53:59
DIRAC group  : dteam_user
path         : /tmp/x509up_u35289
username     : dbauer
properties   : NormalUser
VOMS         : True
VOMS fqan    : ['/dteam']
(base) lxplus739:8.0.0a21 :~] dirac-login gridpp_user -ddd
2022-05-30 12:39:29 UTC Framework [140497768433472] DEBUG: dirac.cfg should be at /afs/cern.ch/work/d/dbauer/public/8.0.0a21/diracos/etc/dirac.cfg
2022-05-30 12:39:29 UTC Framework [140497768433472] DEBUG: CFG merged
2022-05-30 12:39:29 UTC Framework [140497768433472] DEBUG: Updating configuration internals
2022-05-30 12:39:29 UTC Framework [140497768433472] DEBUG: Updating configuration internals
2022-05-30 12:39:30 UTC Framework [140497598961408] DEBUG: Refreshing configuration...
2022-05-30 12:39:30 UTC Framework [140497598961408] DEBUG: Refreshing from list ['https://lbcertifdirac70.cern.ch:9135/Configuration/Server']
2022-05-30 12:39:30 UTC Framework [140497598961408] DEBUG: Randomized server list is https://lbcertifdirac70.cern.ch:9135/Configuration/Server
2022-05-30 12:39:30 UTC Framework [140497598961408] DEBUG:  Trying to refresh from https://lbcertifdirac70.cern.ch:9135/Configuration/Server
2022-05-30 12:39:30 UTC Framework/DIRAC.Core.Tornado.Client.ClientSelector [140497598961408] DEBUG: Trying to autodetect client for https://lbcertifdirac70.cern.ch:9135/Configuration/Server
2022-05-30 12:39:30 UTC Framework/DIRAC.Core.Tornado.Client.ClientSelector [140497598961408] VERBOSE: Using HTTPS for service https://lbcertifdirac70.cern.ch:9135/Configuration/Server
2022-05-30 12:39:30 UTC Framework [140497598961408] DEBUG: Already given a valid url https://lbcertifdirac70.cern.ch:9135/Configuration/Server
2022-05-30 12:39:30 UTC Framework [140497598961408] DEBUG: Already given a valid url https://lbcertifdirac70.cern.ch:9135/Configuration/Server
2022-05-30 12:39:30 UTC Framework [140497598961408] DEBUG: New version available Updating to version 2022-05-25 16:23:15.636599...
2022-05-30 12:39:30 UTC Framework [140497598961408] DEBUG: Updating configuration internals
2022-05-30 12:39:30 UTC Framework [140497598961408] DEBUG: Updated to version 2022-05-25 16:23:15.636599
2022-05-30 12:39:30 UTC Framework [140497768433472] WARN: Could not get NTP time
Parsing command line
Trying to load Resources.LogBackends.StdoutBackend
Trying to load DIRAC.Resources.LogBackends.StdoutBackend
Enter Certificate password: **********
2022-05-30 12:39:35 UTC dirac-login [140497768433472] DEBUG: Trying to load Resources.LogBackends.StdoutBackend
2022-05-30 12:39:35 UTC dirac-login [140497768433472] DEBUG: Trying to load DIRAC.Resources.LogBackends.StdoutBackend
2022-05-30 12:39:35 UTC dirac-login [140497768433472] DEBUG: Refreshing configuration...
2022-05-30 12:39:35 UTC dirac-login [140497768433472] DEBUG: Refreshing from list ['https://lbcertifdirac70.cern.ch:9135/Configuration/Server']
2022-05-30 12:39:35 UTC dirac-login [140497768433472] DEBUG: Randomized server list is https://lbcertifdirac70.cern.ch:9135/Configuration/Server
2022-05-30 12:39:35 UTC dirac-login [140497768433472] DEBUG:  Trying to refresh from https://lbcertifdirac70.cern.ch:9135/Configuration/Server
2022-05-30 12:39:35 UTC dirac-login/DIRAC.Core.Tornado.Client.ClientSelector [140497768433472] DEBUG: Trying to autodetect client for https://lbcertifdirac70.cern.ch:9135/Configuration/Server
2022-05-30 12:39:35 UTC dirac-login/DIRAC.Core.Tornado.Client.ClientSelector [140497768433472] VERBOSE: Using HTTPS for service https://lbcertifdirac70.cern.ch:9135/Configuration/Server
2022-05-30 12:39:35 UTC dirac-login [140497768433472] DEBUG: Already given a valid url https://lbcertifdirac70.cern.ch:9135/Configuration/Server
2022-05-30 12:39:35 UTC dirac-login [140497768433472] DEBUG: Already given a valid url https://lbcertifdirac70.cern.ch:9135/Configuration/Server
2022-05-30 12:39:35 UTC dirac-login/Subprocess [140497768433472] VERBOSE: shellCall: voms-proxy-init -cert /tmp/x509up_u35289 -key /tmp/x509up_u35289 -out /tmp/dbauer/tmpfo6qd413 -voms gridpp:/gridpp -valid 11:54 -vomses /afs/cern.ch/user/d/dbauer/workspace/public/8.0.0a21/diracos/etc/grid-security/vomses -r -timeout 12
2022-05-30 12:39:35 UTC dirac-login [140497768433472] DEBUG: Trying to load Resources.LogBackends.StdoutBackend
2022-05-30 12:39:35 UTC dirac-login [140497768433472] DEBUG: Trying to load DIRAC.Resources.LogBackends.StdoutBackend
2022-05-30 12:39:35 UTC dirac-login [140497768433472] DEBUG: Refreshing configuration...
2022-05-30 12:39:35 UTC dirac-login [140497768433472] DEBUG: Refreshing from list ['https://lbcertifdirac70.cern.ch:9135/Configuration/Server']
2022-05-30 12:39:35 UTC dirac-login [140497768433472] DEBUG: Randomized server list is https://lbcertifdirac70.cern.ch:9135/Configuration/Server
2022-05-30 12:39:35 UTC dirac-login [140497768433472] DEBUG:  Trying to refresh from https://lbcertifdirac70.cern.ch:9135/Configuration/Server
2022-05-30 12:39:35 UTC dirac-login/DIRAC.Core.Tornado.Client.ClientSelector [140497768433472] DEBUG: Trying to autodetect client for https://lbcertifdirac70.cern.ch:9135/Configuration/Server
2022-05-30 12:39:35 UTC dirac-login/DIRAC.Core.Tornado.Client.ClientSelector [140497768433472] VERBOSE: Using HTTPS for service https://lbcertifdirac70.cern.ch:9135/Configuration/Server
2022-05-30 12:39:35 UTC dirac-login [140497768433472] DEBUG: Already given a valid url https://lbcertifdirac70.cern.ch:9135/Configuration/Server
2022-05-30 12:39:35 UTC dirac-login [140497768433472] DEBUG: Already given a valid url https://lbcertifdirac70.cern.ch:9135/Configuration/Server
2022-05-30 12:39:36 UTC dirac-login/DIRAC.Core.Tornado.Client.ClientSelector [140497768433472] DEBUG: Trying to autodetect client for Framework/ProxyManager
2022-05-30 12:39:36 UTC dirac-login/DIRAC.Core.Tornado.Client.ClientSelector [140497768433472] DEBUG: URL resolved: https://lbcertifdirac70.cern.ch:8443/Framework/ProxyManager
2022-05-30 12:39:36 UTC dirac-login/DIRAC.Core.Tornado.Client.ClientSelector [140497768433472] VERBOSE: Using HTTPS for service Framework/ProxyManager
2022-05-30 12:39:36 UTC dirac-login [140497768433472] DEBUG: Discovering URL for service Framework/ProxyManager -> https://lbcertifdirac70.cern.ch:8443/Framework/ProxyManager
2022-05-30 12:39:36 UTC dirac-login [140497768433472] DEBUG: Discovering URL for service Framework/ProxyManager -> https://lbcertifdirac70.cern.ch:8443/Framework/ProxyManager
2022-05-30 12:39:36 UTC dirac-login [140497768433472] NOTICE: Upload proxy to server.
2022-05-30 12:39:36 UTC dirac-login/DIRAC.Core.Tornado.Client.ClientSelector [140497768433472] DEBUG: Trying to autodetect client for Framework/ProxyManager
2022-05-30 12:39:36 UTC dirac-login/DIRAC.Core.Tornado.Client.ClientSelector [140497768433472] DEBUG: URL resolved: https://lbcertifdirac70.cern.ch:8443/Framework/ProxyManager
2022-05-30 12:39:36 UTC dirac-login/DIRAC.Core.Tornado.Client.ClientSelector [140497768433472] VERBOSE: Using HTTPS for service Framework/ProxyManager
2022-05-30 12:39:36 UTC dirac-login [140497768433472] DEBUG: Discovering URL for service Framework/ProxyManager -> https://lbcertifdirac70.cern.ch:8443/Framework/ProxyManager
2022-05-30 12:39:36 UTC dirac-login [140497768433472] DEBUG: Discovering URL for service Framework/ProxyManager -> https://lbcertifdirac70.cern.ch:8443/Framework/ProxyManager
2022-05-30 12:39:36 UTC dirac-login/DIRAC.Core.Tornado.Client.ClientSelector [140497768433472] DEBUG: Trying to autodetect client for Framework/ProxyManager
2022-05-30 12:39:36 UTC dirac-login/DIRAC.Core.Tornado.Client.ClientSelector [140497768433472] DEBUG: URL resolved: https://lbcertifdirac70.cern.ch:8443/Framework/ProxyManager
2022-05-30 12:39:36 UTC dirac-login/DIRAC.Core.Tornado.Client.ClientSelector [140497768433472] VERBOSE: Using HTTPS for service Framework/ProxyManager
2022-05-30 12:39:36 UTC dirac-login [140497768433472] DEBUG: Discovering URL for service Framework/ProxyManager -> https://lbcertifdirac70.cern.ch:8443/Framework/ProxyManager
2022-05-30 12:39:36 UTC dirac-login [140497768433472] DEBUG: Discovering URL for service Framework/ProxyManager -> https://lbcertifdirac70.cern.ch:8443/Framework/ProxyManager
2022-05-30 12:39:36 UTC dirac-login [140497768433472] DEBUG: Trying to load Resources.LogBackends.StdoutBackend
2022-05-30 12:39:36 UTC dirac-login [140497768433472] DEBUG: Trying to load DIRAC.Resources.LogBackends.StdoutBackend
2022-05-30 12:39:36 UTC dirac-login [140497768433472] DEBUG: Refreshing configuration...
2022-05-30 12:39:36 UTC dirac-login [140497768433472] DEBUG: Refreshing from list ['https://lbcertifdirac70.cern.ch:9135/Configuration/Server']
2022-05-30 12:39:36 UTC dirac-login [140497768433472] DEBUG: Randomized server list is https://lbcertifdirac70.cern.ch:9135/Configuration/Server
2022-05-30 12:39:36 UTC dirac-login [140497768433472] DEBUG:  Trying to refresh from https://lbcertifdirac70.cern.ch:9135/Configuration/Server
2022-05-30 12:39:36 UTC dirac-login/DIRAC.Core.Tornado.Client.ClientSelector [140497768433472] DEBUG: Trying to autodetect client for https://lbcertifdirac70.cern.ch:9135/Configuration/Server
2022-05-30 12:39:36 UTC dirac-login/DIRAC.Core.Tornado.Client.ClientSelector [140497768433472] VERBOSE: Using HTTPS for service https://lbcertifdirac70.cern.ch:9135/Configuration/Server
2022-05-30 12:39:36 UTC dirac-login [140497768433472] DEBUG: Already given a valid url https://lbcertifdirac70.cern.ch:9135/Configuration/Server
2022-05-30 12:39:36 UTC dirac-login [140497768433472] DEBUG: Already given a valid url https://lbcertifdirac70.cern.ch:9135/Configuration/Server
2022-05-30 12:39:36 UTC dirac-login [140497768433472] NOTICE: subject      : /C=UK/O=eScience/OU=Imperial/L=Physics/CN=daniela bauer/CN=34857560/CN=4075276226/CN=2933716076
issuer       : /C=UK/O=eScience/OU=Imperial/L=Physics/CN=daniela bauer/CN=34857560/CN=4075276226
identity     : /C=UK/O=eScience/OU=Imperial/L=Physics/CN=daniela bauer
timeleft     : 11:53:58
DIRAC group  : gridpp_user
path         : /tmp/x509up_u35289
username     : dbauer
properties   : NormalUser
VOMS         : True
VOMS fqan    : ['/gridpp']
```

**Comment by TaykYoku** [comment]
Hi, I found the reason that when running the script, before creating the proxy certificate, there is an attempt to synchronize with the configuration configured by TornadoClient. In the next PR(#6138), I turned off syncing until a proxy was created.
